src/editors/EditorContainer.tsx

```diff
@@ -8,9 +8,9 @@ interface Props {
   /** Course ID or Library ID */
   learningContextId: string;
   /** Event handler for when user cancels out of the editor page */
-  onClose?: () => void;
+  onClose?: (prevPath?: string) => void;
   /** Event handler called after when user saves their changes using an editor */
-  afterSave?: () => (newData: Record<string, any>) => void;
+  afterSave?: (prevPath?: string) => (newData: Record<string, any>) => void;
```

Comment:
FYI this component has been refactored with https://github.com/openedx/frontend-app-authoring/pull/1290, so you'll get some conflicts here.